app/controllers/api/registrations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Api::RegistrationsController < DeviseTokenAuth::RegistrationsController
+  def create
+    customer = Stripe::Customer.list(email: params[:email]).data.first
+    customer ||= Stripe::Customer.create({ email: params[:email], source: params[:stripeToken] })
```

Comment:
What is the pipes doing here?

Reply:
It's solar graph changing the word unless, its checking if the first customer returns a result and if not continues on with creating one.

Reply:
Allright! Was confused here 👍

Reply:
it is the same as typing 'unless customer' at the end of line 4. Solargraph puts those pipes. We do it to make so the line 4 is not executed if customer on line 3 if not nil